packages/gg_images/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=gg_images
+pkgver=20.fe35153
+pkgrel=1
+pkgdesc='GG profile photo downloader'
+arch=('x86_64' 'aarch64')
+groups=('blackarch')
```

Comment:
@Jagroop-Singh this is the main group. this is ok and needed unless it's a big/huge prog with huge dependencies (for example: `vmcloak`, which installs virtualbox ...). can you please add a primary group after 'blackarch'? i think 'blackarch-social' and/or 'blackarch-misc' would fit here. not sure though.